src/olympia/versions/models.py

```diff
@@ -247,6 +249,13 @@ def from_upload(cls, upload, addon, selected_apps, channel,
         storage.delete(upload.path)
         version_uploaded.send(sender=version)
 
+        if waffle.switch_is_active('enable-uploads-commit-to-git-storage'):
```

Comment:
Not creating the waffle-switch in the database yet because it's not ready to be activated on dev/stage anyway so I'd create the switches when necessary.